### PR TITLE
Update README.adoc

### PR DESCRIPTION
**Pull request by meswan**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -71,7 +71,7 @@ include::{common-includes}/gitclone.adoc[]
 
 == Setting up MongoDB
 
-This guide uses Docker to run an instance of `MongoDB`. A multi-stage Dockerfile is provided for you. This Dockerfile uses the `mongo` image as the base image of the final stage and gathers the required configuration files. The resulting `mongo` image runs in a Docker container, and you must set up a new database for the microservice. Lastly, the truststore that's generated in the Docker image is copied from the container and placed into the Open Liberty server.
+This guide uses Docker to run an instance of MongoDB. A multi-stage Dockerfile is provided for you. This Dockerfile uses the `mongo` image as the base image of the final stage and gathers the required configuration files. The resulting `mongo` image runs in a Docker container, and you must set up a new database for the microservice. Lastly, the truststore that's generated in the Docker image is copied from the container and placed into the Open Liberty server.
```
Comment (AustinSeto) [inline README.adoc:74]: Paragraph should be separated into multiple lines.

File: README.adoc [hunk 2]
```diff
@@ -168,11 +168,11 @@ server.xml
...
+The features that are required to create the CDI producer for MongoDB are `cdi-2.0`, `ssl-1.0`, `mpConfig-1.4`, and `passwordUtilities-1.0`, which are all specified in the [hotspot=featureManager file=0]`featureManager` element. The Secure Sockets Layer (SSL) context is configured in the [hotspot file=0]`server.xml` file so that the application can connect to MongoDB with TLS. The [hotspot=keyStore file=0]`keyStore` element is configured to point to the `truststore.p12` keystore file that was created in one of the earlier sections. The [hotspot=ssl file=0]`ssl` element is configured to use the `defaultKeyStore` as the keystore and `outboundTrustStore` as the truststore.
```
Comment (AustinSeto) [inline README.adoc:173]: Should break up paragraph into multiple lines

File: README.adoc [hunk 3]
```diff
@@ -241,15 +241,15 @@ The [hotspot=createMongo file=0]`createMongo()` producer method returns an insta
...
+You are going to implement the basic `CRUD` operations, which are create, read, update, and delete. The [hotspot=mongoImports1]`com.mongodb.client` and [hotspot=mongoImports2]`com.mongodb.client.result` packages are used to help implement these `CRUD` operations for the microservice. For more information about these packages, see the https://mongodb.github.io/mongo-java-driver/3.12/javadoc/com/mongodb/client/package-summary.html[com.mongodb.client^] and https://mongodb.github.io/mongo-java-driver/3.12/javadoc/com/mongodb/client/result/package-summary.html[com.mongodb.client.result^] Javadoc. For more information about creating a RESTful service with JAX-RS, JSON-B, and Open Liberty, see the https://openliberty.io/guides/rest-intro.html[Creating a RESTful web serivce^] guide.
```
Comment (AustinSeto) [inline README.adoc:252]: Should break up paragraph into multiple lines

File: README.adoc [hunk 4]
```diff
@@ -275,9 +275,9 @@ Go to the http://localhost:9080/openapi/ui/[^] URL to see the OpenAPI UI that pr
 
 === Implementing the create operation
 
-The [hotspot=add]`add()` method handles the implementation of the `create` operation. An instance of `MongoCollection` is retrieved with the [hotspot=getCollection]`MongoDatabase.getCollection()` method. The `Document` type parameter specifies that the `Document` type is used to store data in the `MongoCollection`. Each crew member is converted into a [hotspot=crewMemberCreation]`Document`, and the [hotspot=insertOne]`MongoCollection.insertOne()` method inserts a new crew member `document`.
+The [hotspot=add]`add()` method handles the implementation of the create operation. An instance of `MongoCollection` is retrieved with the [hotspot=getCollection]`MongoDatabase.getCollection()` method. The `Document` type parameter specifies that the `Document` type is used to store data in the `MongoCollection`. Each crew member is converted into a [hotspot=crewMemberCreation]`Document`, and the [hotspot=insertOne]`MongoCollection.insertOne()` method inserts a new crew member `document`.
```
Comment (AustinSeto) [inline README.adoc:278]: Break up into multiple lines

File: README.adoc [hunk 5]
```diff
@@ -307,9 +307,9 @@ The `\<<ID>>` that you receive is a unique identifier in the collection. Save th
 
 === Implementing the read operation
 
-The [hotspot=retrieve]`retrieve()` method handles the implementation of the `read` operation. The `Crew` collection is retrieved with the [hotspot=getCollectionRead]`MongoDatabase.getCollection()` method. Then, the [hotspot=find]`MongoCollection.find()` method retrieves a `FindIterable` object. This object is iterable for all the crew members documents in the collection, so each crew member document is concatenated into a String array and returned.
+The [hotspot=retrieve]`retrieve()` method handles the implementation of the read operation. The `Crew` collection is retrieved with the [hotspot=getCollectionRead]`MongoDatabase.getCollection()` method. Then, the [hotspot=find]`MongoCollection.find()` method retrieves a `FindIterable` object. This object is iterable for all the crew members documents in the collection, so each crew member document is concatenated into a String array and returned.
```
Comment (AustinSeto) [inline README.adoc:310]: Break up paragraph into multiple lines.

Also, not sure if describing the implementation of the iterable is needed. I think it should be fine to just say the object is iterable for all the crew member documents, and leave it at that.

File: README.adoc [hunk 6]
```diff
@@ -328,9 +328,9 @@ You will receive a response that contains an array of all the members in your cr
...
+The [hotspot=update]`update()` method handles the implementation of the update operation. After `Crew` collection is retrieved, a document is created with the specified object id to query the collection. Next, a new crew member [hotspot=crewMemberUpdate]`Document` is created with the updated configuration. The [hotspot=replaceOne]`MongoCollection.replaceOne()` method is called with the query and new crew member document, which updates all matching queries with the new document. Because the object id is unique in the collection, only one document is updated. `The MongoCollection.replaceOne()` method also returns an UpdateResult, which determines how many documents matched the query. If the match count is zero, then the object id doesn't exist.
```
Comment (AustinSeto) [inline README.adoc]: Break into multiple lines. 

Also not sure if we should put `id` in backticks.

File: README.adoc [hunk 7]
```diff
@@ -358,9 +358,9 @@ You will receive a response that contains the JSON object of the updated crew me
 
 === Implementing the delete operation
 
-The [hotspot=remove]`remove()` method handles the implementation of the `delete` operation. After the `Crew` collection is retrieved, a [hotspot=queryDelete]`document` is created with the specified object id to query the collection. Because the object id is unique in the collection, only one document is deleted. After the document is deleted, the [hotspot=deleteOne]`MongoCollection.deleteOne()` method returns a `DeleteResult`, which determines how many documents were deleted. If the deleted count is zero, then the object id doesn't exist.
+The [hotspot=remove]`remove()` method handles the implementation of the delete operation. After the `Crew` collection is retrieved, a [hotspot=queryDelete]`document` is created with the specified object id to query the collection. Because the object id is unique in the collection, only one document is deleted. After the document is deleted, the [hotspot=deleteOne]`MongoCollection.deleteOne()` method returns a `DeleteResult`, which determines how many documents were deleted. If the deleted count is zero, then the object id doesn't exist.
```
Comment (AustinSeto) [inline README.adoc]: Break into multiple lines. 

Also not sure if we should add backticks to `id` as its a property.

**Review by AustinSeto — DISMISSED** [review]

File: README.adoc (hunk 1 above)
Comment (meswan) [reply README.adoc:74]: Will check with ID

File: README.adoc (hunk 2 above)
Comment (meswan) [reply README.adoc:173]: Will check with ID

File: README.adoc (hunk 3 above)
Comment (meswan) [reply README.adoc:252]: Will check with ID

File: README.adoc (hunk 4 above)
Comment (meswan) [reply README.adoc:278]: Will check with ID

File: README.adoc (hunk 5 above)
Comment (meswan) [reply README.adoc:310]: Will check with ID

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**